### PR TITLE
added descriptions for highlights

### PR DESCRIPTION
**Pull request by niraymak**
## Description

A few sentences describing the overall goals of the pull request's commits.

## Type of change

-   [ ] Bug fix (non-breaking change which fixes an issue)
-   [x] New feature (non-breaking change which adds functionality)
-   [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
-   [ ] This change requires a documentation update

## Checklist

-   [x] My code follows the style guidelines of this project
-   [x] I did not update API Controllers, if I did, I added/changed Postman tests
-   [x] I have performed a self-review of my own code
-   [ ] I have commented my code, particularly in hard-to-understand areas
-   [ ] I have made corresponding changes to the documentation
-   [ ] I have added tests that prove my fix is effective or that my feature works
-   [ ] New and existing unit tests pass locally with my changes
-   [x] I updated the changelog with an end-user readable description
-   [ ] I assigned this pull request to the correct project board to update the sprint board

## Steps to Test or Reproduce

create highlight
update highlight

## Link to issue

Closes: #218

### CODE REVIEW

File: Data/4_Data.csproj [hunk 1]
```diff
@@ -5,12 +5,19 @@
     <LangVersion>8</LangVersion>
   </PropertyGroup>
 
+  <ItemGroup>
+    <Compile Remove="Migrations\20200903084647_AddDescriptionToHighlight.cs" />
+    <Compile Remove="Migrations\20200903084647_AddDescriptionToHighlight.Designer.cs" />
+    <Compile Remove="Migrations\20200904100315_ChangedHighLightDescriptionToRequired.cs" />
+    <Compile Remove="Migrations\20200904100315_ChangedHighLightDescriptionToRequired.Designer.cs" />
+  </ItemGroup>
```
Comment (Brend-Smits) [inline Data/4_Data.csproj:13]: These are still leftovers from testing your migrations. This code block can be removd.

**Review by Brend-Smits — CHANGES_REQUESTED** [review]
Welcome to the team Niray!
Looks great, thanks for also updating the postman tests. 
Please add an entry to the changelog though!

File: Data/4_Data.csproj (hunk 1 above)
Comment (Brend-Smits) [reply Data/4_Data.csproj]: If you have fixed this comment, click on "Resolve conversation"!

**Review by Brend-Smits — APPROVED** [review]
Next time please also include a description in the pull request (all the way on the top of the template). This is so that people that were not involved in the development process can easily understand what this is about without wasting too much time :)

File: Data/4_Data.csproj (hunk 1 above)
Comment (niraymak) [reply Data/4_Data.csproj]: @Brend-Smits Isn't it already done? It shows "Outdated" next to the file name. :)

Comment (Brend-Smits) [reply Data/4_Data.csproj]: Nope, see: https://gyazo.com/85a5a5329ddc447f985727d1d3457aba
Bottom left. I can of course do it myself but I'm trying to teach you all the processes! ^.^

**Comment by Brend-Smits** [comment]
@niraymak It looks like there is a merge conflict at this moment. It's an easy one to fix but always good practice.
I recommend merging Develop into this branch to ensure that the new line endings are enforced properly.

File: Data/4_Data.csproj [hunk 2]
```diff
@@ -7,10 +7,10 @@
...
  <ItemGroup>
     <PackageReference Include="Bogus" Version="29.0.1" />
-    <PackageReference Include="Microsoft.EntityFrameworkCore" Version="3.1.2" />
-    <PackageReference Include="Microsoft.EntityFrameworkCore.Relational" Version="3.1.2" />
-    <PackageReference Include="Microsoft.EntityFrameworkCore.SqlServer" Version="3.1.2" />
-    <PackageReference Include="Microsoft.EntityFrameworkCore.Tools" Version="3.1.2">
+    <PackageReference Include="Microsoft.EntityFrameworkCore" Version="3.1.3" />
+    <PackageReference Include="Microsoft.EntityFrameworkCore.Relational" Version="3.1.3" />
+    <PackageReference Include="Microsoft.EntityFrameworkCore.SqlServer" Version="3.1.3" />
+    <PackageReference Include="Microsoft.EntityFrameworkCore.Tools" Version="3.1.3">
```
Comment (Maxvanhattum) [inline Data/4_Data.csproj:13]: Don't know if we can just update packages, not familiar enough with the codebase. But is probably fine.

File: Data/Migrations/20200904113526_DecriptionAddedForHighlightTable.cs [hunk 3]
```diff
@@ -0,0 +1,62 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
```
Comment (Maxvanhattum) [inline Data/Migrations/20200904113526_DecriptionAddedForHighlightTable.cs:1]: Are these generated by efcore migration tool? Not familiar enough with this to review it.

File: Data/Migrations/ApplicationDbContextModelSnapshot.cs [hunk 4]
```diff
@@ -150,7 +154,7 @@ protected override void BuildModel(ModelBuilder modelBuilder)
                         .HasColumnType("int")
                         .HasAnnotation("SqlServer:ValueGenerationStrategy", SqlServerValueGenerationStrategy.IdentityColumn);
 
-                    b.Property<int?>("RoleId")
```
Comment (Maxvanhattum) [inline Data/Migrations/ApplicationDbContextModelSnapshot.cs:153]: Might cause an issue if we change a nullable to not nullable. But again I'm not familiar with migrations and if this is migrations doing its magic it should be good.

File: Postman/dex.postman_collection.json [hunk 5]
```diff
@@ -1,6 +1,6 @@
 {
```
Comment (Maxvanhattum) [inline Postman/dex.postman_collection.json:1]: Not familiar with postman tests, can't review this. I'll make sure to look into it for the future

**Review by Maxvanhattum — APPROVED** [review]
Made some comments in the files, it all looks good but some are just remarks which are me not being comfortable with the used technology and asking some questions.
Everything ran on my local environment, logged in for swagger with oauth and tested main features (not the ones needing higher auth scopes)

File: Data/Migrations/20200904113526_DecriptionAddedForHighlightTable.cs (hunk 3 above)
Comment (niraymak) [reply Data/Migrations/20200904113526_DecriptionAddedForHighlightTable.cs:1]: Yes, if any changes are made in the database (models) you need to create a migration. This tool is used to create it.

You can find some documentation about it here:
https://docs.microsoft.com/en-us/ef/core/managing-schemas/migrations/?tabs=dotnet-core-cli

File: Data/Migrations/ApplicationDbContextModelSnapshot.cs (hunk 4 above)
Comment (niraymak) [reply Data/Migrations/ApplicationDbContextModelSnapshot.cs:153]: To be honest i'm not sure why this is changed. @Brend-Smits do you know why? Should I revert it?

Comment (Brend-Smits) [reply Data/Migrations/ApplicationDbContextModelSnapshot.cs:153]: No, keep it. I mentioned this earlier but I think we forgot about a migration earlier. Good question!

**Pull request merged**